### PR TITLE
Update community-examples.json

### PR DESCRIPTION
**Pull request by igorbakman**
After not finding an example to the use of DynamoDB with AWS-Lambda, I've decided to add my implementation for future reference.

### CODE REVIEW

**Comment by igorbakman** [comment]
What is the problem?

**Comment by christophgysin** [comment]
There were linting issues on master, fixed in #194. Please rebase.

Also, when updating community-examples.json, remember to run:

```bash
$ npm run docs
```

**Comment by igorbakman** [comment]
can you please check my latest commit?

**Comment by christophgysin** [comment]
Please rebase on top of current master.

**Comment by igorbakman** [comment]
so can I close it?

**Comment by christophgysin** [comment]
If you close it, you discard it. You'll need to wait for someone with write access to merge it.

File: community-examples.json [hunk 1]
```diff
@@ -272,5 +272,10 @@
     "name": "serverless + medium text-to-speech",
     "description": "Serverless-based, text-to-speech service for Medium articles",
     "githubUrl": "https://github.com/RafalWilinski/serverless-medium-text-to-speech"
+  },
+  {
+    "name": "serverless + java DynamoDB imlementation example",
+    "description": " example for java programmers that want to work with AWS-Lambda and DynamoDB",
```
Comment (christophgysin) [inline community-examples.json]: Could you remove the extra whitespace and fix capitalization?

**Pull request merged**